docs/element-definitions.md

```diff
@@ -367,21 +418,20 @@ The example below defines an Element matching only the null value.
 ### Boolean Element
 
 Type with domain of two values: true and false.
-`content` property MUST contain a boolean value if set.
 
 #### Template
 - `element` - `"boolean"`
 - `attributes`
   - `typeAttributes` ([Array](#array-element)[[String](#string-element)])
-    - `fixed` ([String](#string-element)) - domain restricted to value given in `content`
+    - `fixed` ([String](#string-element)) - The type this Element describes is restricted to the value given in `content`
   - `validation` - _reserved for future use_
-  - `samples` ([Array[Boolean]](#array-element)) - array of alternative sample values for this Element; type of items in `samples` MUST match the type of this Element
-  - `default` (Boolean Element) - default value for this Element; type of `default` MUST match the type of this Element
-- `content` - ⊥ (false) or ⊤ (true)
+  - `samples` ([Array[[Boolean](#boolean-element)]](#array-element)) - Alternative sample values for this Element; type of items in `samples` MUST match the type this Element describes
+  - `default` ([Boolean](#boolean-element)) - Default value for this Element; type of `default` MUST match the type this Element describes
+- `content` - _false_ or _true_
 
 #### Example
 
-Type Element matching only boolean values (JSON `true`, `false`):
+Type Element representing only boolean values (JSON `true`, `false`):
```

Comment:
Perhaps this wording can be unclear, because no value is also permitted, not just `true` or `false`. There can be the lack of a value which tooling will have to support.

```json
{
    "element": "boolean"
}
```

Reply:
`{"element": "boolean"}` really matches only *true* or *false*. The lack of value can be represented through *nullable*.

Reply:
I think I mis-understood the context of this change, I was referring to what values are permitted in `content` of the boolean element. 

> The lack of value can be represented through nullable.

Yes `{"element": "boolean"}` only permits `true` or `false` as values. However even without nullable, we can have an boolean element which doesn't have a default, samples or value. The serialised element will NOT contain true or false in this case but lack of a value. This is not to say that null is permitted (lack of nullable) but there isn't any values.

I don't think there is anything to change here, looking deeper this sentence is referring to a particular example. I think I mis-understood when originally reading.